test/HostActivationTests/GivenThatICareAboutLightupAppActivation.cs

```diff
@@ -0,0 +1,188 @@
+
+            // Delete any existing artifacts
+            if (Directory.Exists(customLightupPath))
+            {
+                Directory.Delete(customLightupPath, true);
+            }
+
+            customLightupPath = Path.Combine(customLightupPath, "Microsoft.NETCore.App");
+            customLightupPath = Path.Combine(customLightupPath, builtSharedFXVersion);
+
+            // Create the folder to which lightup.deps.json will be copied to.
+            Directory.CreateDirectory(customLightupPath);
+            
+            // Copy the lightup.deps.json
+            var libDepsJson = fixtureLib.TestProject.DepsJson;
+            File.Copy(libDepsJson, Path.Combine(customLightupPath, Path.GetFileName(libDepsJson)));
+
+            // Copy the library to the location of the lightup app (app-local)
+            var destLibPath = Path.Combine(Path.GetDirectoryName(appDll), Path.GetFileName(libDll));
+            File.Copy(libDll, destLibPath);
+
+            // Execute the test using the custom lightup path where lightup.deps.json can be found.
+            dotnet.Exec("exec", "--additional-deps", baseDir, appDll)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdOutContaining("Hello LightupClient");
+        }
+
+        // Attempt to run the app without lightup deps.json specified but lightup library present in the expected 
+        // probe location (of being app-local).
+        [Fact]
+        public void Muxer_activation_of_LightupApp_WithLightupLib_NoLightupDepsJson_Fails()
+        {
+            var fixtureLib = PreviouslyBuiltAndRestoredLightupLibTestProjectFixture
+                .Copy();
+
+            var fixtureApp = PreviouslyBuiltAndRestoredLightupAppTestProjectFixture
+                .Copy();
+
+            var dotnet = fixtureApp.BuiltDotnet;
+            var appDll = fixtureApp.TestProject.AppDll;
+            var libDll = fixtureLib.TestProject.AppDll;
+
+            var destLibPath = Path.Combine(Path.GetDirectoryName(appDll), Path.GetFileName(libDll));
+
+            // Copy the library to the location of the lightup app
+            File.Copy(libDll, destLibPath);
+
+            dotnet.Exec("exec", appDll)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute(fExpectedToFail:true)
+                .Should()
+                .Fail()
+                .And
+                .HaveStdOutContaining("Exception: Failed to load the lightup assembly!");
+        }
+
+        /*
+         * This method is needed to workaround dotnet build not placing the host from the package
+         * graph in the build output.
+         * https://github.com/dotnet/cli/issues/2343
+         */
+        private static void ReplaceTestProjectOutputHostInTestProjectFixture(TestProjectFixture testProjectFixture)
```

Comment:
This method doesn't appear to be used.  Plus it is duplicated with [https://github.com/dotnet/core-setup/blob/492bb0cc31e1a85affa6e8758466af5cdd798d09/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs#L151](https://github.com/dotnet/core-setup/blob/492bb0cc31e1a85affa6e8758466af5cdd798d09/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs#L151)